app/controllers/landlords_controller.rb

```diff
@@ -0,0 +1,47 @@
+class LandlordsController < ApplicationController
+  def create
+  	landlord = Landlord.new(landlord_params)
+  	@error = false
+  	if landlord.save!
+  		redirect_to landlords_show_url
+  	else
+  		@error = true
+  		redirect_to landlords_show_url
+	  end
+  end
+
+  def show
+  	if user_signed_in? and current_user.type == 'Landlord'
+  	  @landlord = current_user
+  	  @properties = @landlord.properties
+  	end
+  end
+
+  def update
+  	@error = false
+  	if params[:id] == current_user.id
+  	  @landlord = Landlord.find(params[:id])
+  	  @landlord.update!
```

Comment:
the check should be on the update for the error since we will be printing out the users errors, you could do something like puts `current_user.errors.full_messages`